tests/unit/test_utils.py

```diff
@@ -434,3 +434,15 @@ def test_sort(self):
         data = dict(y=2, x=1)
         assert fmf.utils.dict_to_yaml(data) == "y: 2\nx: 1\n"
         assert fmf.utils.dict_to_yaml(data, sort=True) == "x: 1\ny: 2\n"
+
+
+class TestBETTERNAME:
+    def test_invalid(self):
+        for s in ["", "/", ";/;", ";;;", "/a/b/c", "/a/b/c/d",
```

Comment:
Quick comment, how about splitting this list line-by-line so we can add comment on what is caught in each part

Reply:
updated